ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIComponent/UISwitch+ToDoGardenSwitch.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UISwitch+ToDoGardenSwitch.swift
+//
+//
+//  Created by SONG on 3/1/24.
+//
+
+import ToDoGardenUIResource
+import UIKit.UISwitch
+
+extension UISwitch {
+  public func toDoGardenSwitchStyle(isOn: Bool) {
+    self.isOn = isOn
+    self.setupOnTintColor()
+    self.setupOffTintColor()
+  }
+}
+
+// MARK: - private functions
+
+extension UISwitch {
+  private func setupOnTintColor() {
+    self.onTintColor = UIColor.toDoGardenGreenDark
+  }
+  
+  private func setupOffTintColor() {
+    self.subviews.first?.subviews.first?.backgroundColor = UIColor.toDoGardenGreenGray
```

Comment:
```suggestion
    let view = self.subviews.first?.subviews.first
    view?.backgroundColor = UIColor.toDoGardenGreenGray
```

이런식으로 상수에 할당해주면 의미가 조금 더 명확해질 것 같아요 :)

Reply:
+ 앱이 백그라운드 상태에 들어갔다가 다시 돌아오는 경우 해당 색이 잘 반영이 되지 않네요 확인 부탁드립니다!

![Simulator Screen Recording - iPhone 15 Pro - 2024-03-06 at 14 57 29](https://github.com/ToDo-Garden/ToDo-Garden/assets/63908856/cb9c7bc3-d83f-400b-83b7-68f35d509824)

Reply:
- 해당 버그 수정하였습니다 ! 

> [8d462a8](https://github.com/ToDo-Garden/ToDo-Garden/pull/41/commits/8d462a8c18965f3b0c554379a18076c0e3cb889a)

백그라운드에서 포그라운드로 복귀하면, UISwitch의 subview들이 새롭게 대체되는 현상을 발견하였습니다. 

![스크린샷 2024-03-06 오후 5 02 52](https://github.com/ToDo-Garden/ToDo-Garden/assets/88966578/662cdda3-21ee-40fd-b175-801a08458292)
![스크린샷 2024-03-06 오후 5 03 21](https://github.com/ToDo-Garden/ToDo-Garden/assets/88966578/63335289-3316-4e2d-8abf-4f9fe17a6ba2)
(전 , 후의 주소값이 다름) 
즉, 전과 후의 스위치의 배경을 담당하는 UIView는 다른 객체입니다. 

백그라운드에서 포그라운드로 진입시 , subview 들이 새롭게 대체된다면, UISwitch의 layoutSubview()가 호출될 것이라고 판단하여 layoutSubview()를 오버라이딩 하기위해 Extension에서 Class로 재정의 하였습니다.  

아래는 결과 화면이며, 이제 해당 오류현상이 발생하지 않습니다.
![Simulator Screen Recording - iPhone 15 Pro - 2024-03-06 at 17 10 21](https://github.com/ToDo-Garden/ToDo-Garden/assets/88966578/f31ff87f-c90d-464f-a0e3-3546b08f3048)

Reply:
@HG-SONG 
내용 확인했습니다:)